### PR TITLE
Simplify and fix remote write example

### PR DESCRIPTION
**Pull request by juliusv**
After removing gRPC, this can be simplified again. Also, the
configuration for the remote storage moved from flags to the config
file.

### CODE REVIEW

**Comment by juliusv** [comment]
@brian-brazil

File: documentation/examples/remote_storage/server.go [hunk 1]
```diff
@@ -21,43 +21,15 @@ import (
...
-}
-
-type WriteServer interface {
-	Write(context.Context, *remote.WriteRequest) (*remote.WriteResponse, error)
-}
-
-// AppenderHandler returns a http.Handler that accepts proto encoded samples.
-func AppenderHandler(s WriteServer) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	http.HandleFunc("/push", func(w http.ResponseWriter, r *http.Request) {
```
Comment (brian-brazil) [inline documentation/examples/remote_storage/server.go]: Would /receive be better?

**Comment by brian-brazil** [comment]
:+1:

File: documentation/examples/remote_storage/server.go (hunk 1 above)
Comment (juliusv) [inline documentation/examples/remote_storage/server.go]: Changed to /receive.

**Pull request merged**